internal/spokes/spokes.go

```diff
@@ -34,7 +36,7 @@ type SpokesReceivePack struct {
 	output           io.Writer
 	err              io.Writer
 	capabilities     string
-	repoPath         string
+	RepoPath         string
```

Comment:
might be nice to move this to a the top or bottom of the struct, to make it more obvious that its a public member.